custom/build.gradle.kts

```diff
@@ -37,7 +37,7 @@ android {
   }
   splits {
     abi {
-      isUniversalApk = false
+      isUniversalApk = true
```

Comment:
We should reverse this after that task.

Reply:
I have reverted this change because now we are uploading a dummy bundle so does not need a universal APK.